### PR TITLE
[#508] OSP - Infra Mapping Wizard Step 4

### LINKED ISSUE #508: OSP Support: Changes to Infra Mapping Wizard - Step 4
Clearer instructional text

6a-createmapping-networks_2

Network mapping for OSP, no real changes from RHV 
6b-createmapping-networks_2

### PR DESCRIPTION
**Pull request by AllenBW**
fixes #508 

#531 ought to be merged before this, so you can seeeeeeeee da AWESOME 🌮 💃 (but that aside, this is g2g)

### Rhevm
screen shot 2018-08-02 at 10 40 30 am

### Openstack
screen shot 2018-08-07 at 9 36 00 am

### CODE REVIEW

File: app/javascript/react/screens/App/common/networkKey.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
 /**
  * returns a unique network based on current UI model
  */
-export const networkKey = network => `${network.clusterId}\\${network.uid_ems}`;
+export const networkKey = network => `${network.clusterId}\\${network.uid_ems || network.ems_ref}`;
```
Comment (AllenBW) [inline app/javascript/react/screens/App/common/networkKey.js]: Don't freak, rhevm is the same, its just osp providers don't have a uid_ems rather they have an ems_ref

File: app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardNetworksStep/MappingWizardNetworksStepActions.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import {
   FETCH_V2V_TARGET_NETWORKS,
   QUERY_ATTRIBUTES
 } from './MappingWizardNetworksStepConstants';
+import { V2V_TARGET_PROVIDER_NETWORK_KEYS } from '../../MappingWizardConstants';
```
Comment (michaelkro) [inline app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardNetworksStep/MappingWizardNetworksStepActions.js]: I think you need to add this constant still 😛

Comment (AllenBW) [reply app/javascript/react/screens/App/Overview/screens/MappingWizard/components/MappingWizardNetworksStep/MappingWizardNetworksStepActions.js]: yep, that's what i get for working off the branch of a branch 🤦‍♂️ ❤️

**Comment by vconzola** [comment]
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1632355

**Pull request merged**